sdk/core/core-paging/tsconfig.es.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "module": "es6",
```

Comment:
I have copied this file from core-http in https://github.com/Azure/azure-sdk-for-js/pull/11563. It seems we still want ES module, unless the default is no longer CommonJS?

Reply:
ops, my bad.